<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digimon Digital Battle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="digimon.qrc">:/new/Sprites/Digimon/Background.png</pixmap>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>204</width>
       <height>167</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="Player1Mugshot">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="digimon.qrc">:/new/DigiElegidos/Digimon/Davis.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Wins</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="PLayer1Wins">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="Player1Name">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string>Davis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Player1NombreDigimonPlayer1">
          <property name="text">
           <string>Veemon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Player1Digimon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="digimon.qrc">:/new/Digimons/Digimon/Veemon.png</pixmap>
          </property>
          <property name="margin">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>419</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>206</width>
       <height>167</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="Player2Name">
          <property name="text">
           <string>Yolei</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Player2NombreDigimon">
          <property name="text">
           <string>Hawkmon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Player2Digimon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="digimon.qrc">:/new/Digimons/Digimon/Hawkmon.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="Player2Mugshot">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="digimon.qrc">:/new/DigiElegidos/Digimon/Yolei.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Wins</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="Player2Wins">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>419</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>461</width>
       <height>76</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="Player2Attack">
          <property name="text">
           <string>Attack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Player2Defend">
          <property name="text">
           <string>Defend</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="Player2Skip">
          <property name="text">
           <string>Skip</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Player2Withdraw">
          <property name="text">
           <string>Withdraw Round</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QProgressBar" name="Player2Hp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>441</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>461</width>
       <height>76</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="Player1Attack">
          <property name="text">
           <string>Attack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Player1Defend">
          <property name="text">
           <string>Defend</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="Player1Skip">
          <property name="text">
           <string>Skip</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Player1Withdraw">
          <property name="text">
           <string>Withdraw Round</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QProgressBar" name="Player1Hp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>441</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="digimon.qrc"/>
 </resources>
 <connections/>
</ui>
